scripts/migrations/107-add-account-versioning.py

```diff
@@ -0,0 +1,23 @@
+"""
```

Comment:
@bowenwang1996 does this mean that everyone (every RPC node, every Bisontrails instance, every indexer node, etc) will be required to run this script or re-download the genesis file and records once we enable this feature? It feels like quite a cumbersome process. It seems that I had the wrong impression that the genesis file never changes. Do nodes need the file only on their first start and it is never used/loaded beyond that point?

Reply:
This only happens when we do hard fork. Also unless it is mainnet, there is no reason for other people to run the migration themselves and they can wait for us to provide the new genesis. If it is mainnet, then first of all we do not expect to do hardforks there and also even if we do it requires much more careful coordination